spec/active_record/postgres_pub_sub/listener_spec.rb

```diff
@@ -17,7 +19,7 @@
       Thread.new do
         listener_loop(**listener_options)
       ensure
-        base_class.clear_active_connections!
```

Comment:
NOTE: delegation to the connection handler was deprecated in activerecord 7.1 and removed in activerecord 7.2.